students/brian_warn/session03/list_lab.py

```diff
@@ -0,0 +1,81 @@
+#!/Library/Frameworks/Python.framework/Versions/3.6/bin/python
```

Comment:
you rarely want to be this specific with a #! line.

ifyou use:

#!/usr/bin/env python3

Then you will get whatever python3 is on your PATH.